src/wazuh_mcp_server/config_validator.py

```
@@ -0,0 +1,411 @@
+        
+        # Check required variables
+        for var, description in self.required_vars.items():
+            value = os.getenv(var)
+            if not value:
+                errors.append(f"Missing required environment variable: {var} ({description})")
+            elif var == "WAZUH_PASS" and len(value) < 8:
+                warnings.append(f"Weak password for {var}")
+        
+        # Validate configurations
+        try:
+            wazuh_config = WazuhConfig(
```

Ruff (0.12.2)

Local variable `wazuh_config` is assigned to but never used

Remove assignment to unused variable `wazuh_config`

(F841)